bench-tps/src/cli.rs

```diff
@@ -469,14 +478,21 @@ pub fn parse_args(matches: &ArgMatches) -> Result<Config, &'static str> {
     );
     args.websocket_url = websocket_url;
 
+    if matches.is_present("identity") {
+        eprintln!("Warning: --identity is deprecated. Please use --authority");
+    }
+
     let (_, id_path) = ConfigInput::compute_keypair_path_setting(
-        matches.value_of("identity").unwrap_or(""),
+        matches
+            .value_of("authority")
+            .or(matches.value_of("identity"))
+            .unwrap_or(""),
         &config.keypair_path,
     );
     if let Ok(id) = read_keypair_file(id_path) {
         args.id = id;
-    } else if matches.is_present("identity") {
-        return Err("could not parse identity path");
+    } else if matches.is_present("authority") || matches.is_present("authority") {
```

Comment:
Typo?
```suggestion
    } else if matches.is_present("identity") || matches.is_present("authority") {
```

Reply:
Sorry for that, will note that next time. Thanks for review